rholang/src/test/scala/coop/rchain/rholang/interpreter/MatchTest.scala

```diff
@@ -421,4 +423,36 @@ class VarMatcherSpec extends FlatSpec with Matchers {
     val failResult = spatialMatch(failTarget, pattern).runS(emptyMap)
     failResult should be(None)
   }
+
+  "Matching a target with var ref and a pattern with a var ref" should "ignore locallyFree" in {
```

Comment:
Is _ignore_ the right word?

Should this match?
```
new z in {
  new x in { for (=*x <- @Nil) { Nil } } matches new x in for(=*z) <- @Nil) { Nil } } 
}
```
probably not, but _ignore_ suggests that they would (because locallyFree in the patterns (`=*x` and `=*z`) are ignored.

Reply:
Those fail to match because the variable references differ. locallyFree would also differ if we had left it alone.
So ignore is the right word.